caliban/cloud/core.py

```diff
@@ -486,7 +486,7 @@ def build_job_specs(
                     experiments=experiments)
 
 
-def generate_image_tag(project_id, docker_args, dry_run: bool = False):
+def generate_image_tag(project_id, docker_args, dlvm: str = None, dry_run: bool = False):
```

Comment:
if we call it `--base_image` we can add that key to the `generate_docker_args` function here: https://github.com/google/caliban/blob/master/caliban/cli.py#L536